src/app/services/answer.service.ts

```diff
@@ -4,21 +4,23 @@ import { catchError, tap } from 'rxjs/operators';
 import { HttpClient } from '@angular/common/http';
 
 import { Answer } from '../models/answer';
+import { Logger } from './log.service';
 
 @Injectable({
   providedIn: 'root'
 })
 export class AnswerService {
   private readonly _randomId: number = Math.floor(Math.random() * 10) + 1;
   private readonly answerUrl = `api/answers/${this._randomId}`;
+  private log = new Logger();
 
   constructor(private http: HttpClient) { }
 
   getAnswer(): Observable<Answer> {
     return this.http.get<Answer>(this.answerUrl).pipe(
-      tap(() => console.log('fetched answer')),
+      tap(() => this.log.info('fetched answer')),
       catchError(err => {
-        console.error(`getAnswer failed: ${err.message}`);
+        this.log.error(`getAnswer failed: ${err.message}`);
         return of(<Answer>{ id: 1, word: 'ADEPT' });
```

Comment:
I'm a little confused as to why you are returning something in the `catchError`. It looks like you are always going to return `{ id: 1, word: 'ADEPT' }` whenever the HTTP request fails? I think it would be better to show an error message or something. It doesn't have to be real polished, but I think it'd be better than silently failing like this.

Reply:
I think the idea was the player would be able to continue playing offline if the http request failed (albeit, they would only be able to play with the word 'ADEPT'). But I see the benefit in "letting it crash" so I'll go ahead and update!